docs/GHA.md

```diff
@@ -0,0 +1,50 @@
+# GitHub Actions
+
+The Buildkite Migration tool supports transitioning GitHub Action workflow definitions to Buildkite pipelines.
+
+## Partially supported properties
+
+### concurrency
+
+The `concurrency` key within a GitHub Action workflow can be defined at topmost level, and within a single job. The Buildkite Migration tool supports the transition of the `concurrency` key (and `group`/`cancel-in-progress`) in job definitions: and maps to the `concurrency_group` [key](https://buildkite.com/docs/pipelines/controlling-concurrency#concurrency-groups) available within Buildkite. Buildkite also allows a upper limit on how much jobs are created through a single step definition with the `concurrency` key: which is set as `1` by default (there isn't a translatable key within a GitHub Action workflow).
+
+The `cancel-in-progress` Boolean value that can be defined in a `concurrency` hash inside a GitHub Action job workflow maps to the Buildkite pipeline setting of [Cancel Intermediate Builds](https://buildkite.com/docs/pipelines/skipping#cancel-running-intermediate-builds) - which can be set within a pipeline's settings page, or when creating/updating a pipeline via the [REST](https://buildkite.com/docs/apis/rest-api/pipelines#create-a-yaml-pipeline)/[GraphQL](https://buildkite.com/docs/apis/graphql/schemas/mutation/pipelinecreate) APIs.
+
+### env 
+
+GitHub Actions allows the specification of environment variables at the workflow level, at a `job` level, and for each `step` within a `job`.
+
+Environment variables that are defined at the top of a workflow will be transition to [build level](https://buildkite.com/docs/pipelines/environment-variables#environment-variable-precedence) environment variables in the generated Buildkite pipeline. Environment variables defined within the context of each of a workflows' `jobs` are transitioned to [step level](https://buildkite.com/docs/pipelines/environment-variables#runtime-variable-interpolation) environment variables.
```

Comment:
why is this _partially supported_? It sounds like they are translated correctly

Reply:
`workflow's`, rather than` workflows'`; a workflow is a singular.

Reply:
I moved it up to supported: I think I was missing the step level env / variable assignment (along with job level/build level)